addons/website_forum/static/src/js/website_forum.js

```diff
@@ -1,17 +1,94 @@
 /** @odoo-module **/
-
-import { markup } from "@odoo/owl";
+import { Component, useState, xml, markup, onWillStart } from "@odoo/owl";
 import { FlagMarkAsOffensiveDialog } from "../components/flag_mark_as_offensive/flag_mark_as_offensive";
 import { ConfirmationDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
 import { cookie } from "@web/core/browser/cookie";;
 import { loadWysiwygFromTextarea } from "@web_editor/js/frontend/loadWysiwygFromTextarea";
 import publicWidget from "@web/legacy/js/public/public_widget";
 import { session } from "@web/session";
 import { rpc } from "@web/core/network/rpc";
-import { escape } from "@web/core/utils/strings";
+import { get } from "@web/core/network/http_service";
 import { _t } from "@web/core/l10n/translation";
 import { renderToElement } from "@web/core/utils/render";
 import { scrollTo, closestScrollableY } from "@web/core/utils/scrolling";
+import { attachComponent } from "@web_editor/js/core/owl_utils";
+import { SelectMenu } from "@web/core/select_menu/select_menu";
+import { DropdownItem } from "@web/core/dropdown/dropdown_item";
+
+class SelectMenuWrapper extends Component {
+    static template = xml`
```

Comment:
Hello @bvr-odoo , Agree to your point. but don't know, in which case user will xPath this template. May be it's rare case. I don't think there is any need to make it more complicated & confusing by creating separate xml for this small part of code. :face_with_diagonal_mouth: 

But still, If you feel we should do that. let me know, I'll adapt as well.

Thanks : )  :smile:

Reply:
Hello @divy-odoo 
It’s true that in this case, it doesn’t seem useful to allow it to be x-path... So IMO it can remain like that
I let @qsm-odoo decide what to do :wink:

Reply:
@divy-odoo move it to an xml file please :+1: it will also be useful later when we'll lazy load xml definitions when the component is actually used. And I don't think it's more unlikely to xpath this than any other xml? :shrug:

Reply:
Sure @qsm-odoo, moved the xml code to separate file.

Thanks : )